### PR TITLE
DOC-X101 - solar system command update

### PR DESCRIPTION
**Pull request by pharsfalvi**
Just some typo-like updates, hopefully useful.

### CODE REVIEW

File: content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ The project containing the sample CorDapp opens.
 `corda-cli network config docker-compose solar-system`
 
 3. Deploy your network by building the network deployment Docker file using Corda CLI:
-  `corda-cli network deploy -n solar-system -f solar-system.yaml > solar-system-compose`
+  `corda-cli network deploy -n solar-system -f solar-system.yaml > docker-compose.yaml`
```
Comment (driessamyn) [inline content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md:83]: I think this change conflicts with the change in https://github.com/corda/corda-docs-portal/pull/275

File: content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ This command also specifies the output file. In this case, the output file is `c
 
 In Corda 4, this process was much more involved. Now you can install the application on the network with a single command:
 
-  `corda-cli package install -n solar-system cordaSolarSystem.cpb`
+  `corda-cli package install -n solar-system corda5-hello-solarsystem.cpb`
```
Comment (driessamyn) [inline content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md:122]: I'm pretty sure the cpb will be found in one of the build folders. We'd need to check the gradle config to see where the cpb is output and what it's called.

**Comment by taylorharvey-r3** [comment]
As @driessamyn mentioned above, we have a fix for these issues in https://github.com/corda/corda-docs-portal/pull/275 so I will close PR #277  . Thank you for your contribution, @pharsfalvi ! It's very useful to have this feedback.

**Comment by pharsfalvi** [comment]
@taylorharvey-r3  thanks a lot for the quick review :thumbsup: 
As this PR is closed now, I added subsequent points to consider as a review on #275.

File: content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md (hunk 2 above)
Comment (pharsfalvi) [reply content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md:122]: Yes, indeed you get the workflow and the contract packages under the corresponding build/lib folders, however both the package and the workflow packages (`.cpk`) are contained in a bundle (`.cpb`) as a result of Step 5, which demonstrates the `cordapp-builder` CLI utility.

**Pull request closed without merging**